src/app/accounts/two-factor.component.html

```diff
@@ -138,7 +138,7 @@
                 aria-hidden="true"
               ></i>
             </button>
-            <a routerLink="/" class="btn btn-outline-secondary btn-block ml-2 mt-0">
+            <a routerLink="/login" class="btn btn-outline-secondary btn-block ml-2 mt-0">
```

Comment:
Commenting on this for documentation. You shouldn't link to `/` because that landing location is now the `HomeGuard`, which always redirects. You can get stuck in a redirect loop where angular refuses to continue loading.